quickevent/app/plugins/Event/src/eventplugin.cpp

```diff
@@ -696,13 +699,12 @@ void EventPlugin::connectToSqlServer()
 		}
 	}
 	setSqlServerConnected(connect_ok);
-	m_actCreateEvent->setEnabled(connect_ok);
-	m_actOpenEvent->setEnabled(connect_ok);
-	m_actEditEvent->setEnabled(connect_ok);
-	m_actExportEvent_qbe->setEnabled(connect_ok);
-	m_actImportEvent_qbe->setEnabled(connect_ok);
+	m_actEvent->setEnabled(connect_ok);
+	m_actExport->setEnabled(connect_ok);
+	m_actImport->setEnabled(connect_ok);
```

Comment:
disable menu completely  if not connected to a database
only `Connect to database` and `Quit` options are left